projects/plutus/test/test_crc15.c

```diff
@@ -105,3 +105,17 @@ void test_crc15_calculate_rdcomm_slave_result() {
 
   TEST_ASSERT_EQUAL(pec, crc15_calculate(data, SIZEOF_ARRAY(data)));
 }
+
+void test_crc15_calculate_readcfg() {
```

Comment:
Functions that don't take any parameters should be `ret_type foo(void)`. https://stackoverflow.com/questions/693788/is-it-better-to-use-c-void-arguments-void-foovoid-or-not-void-foo